faer-core/src/lib.rs

```diff
@@ -4673,7 +5150,7 @@ impl<'a, E: Entity> MatMut<'a, E> {
     /// Fills the elements of `self` with copies of `constant`.
     #[inline(always)]
     #[track_caller]
-    pub fn set_constant(&mut self, constant: E) {
```

Comment:
An absolute nitpick, but how about naming this method just `fill` as analogous to [`slice::fill`](https://doc.rust-lang.org/nightly/std/primitive.slice.html#method.fill)? As for `fill_with_zero`, it does not have a counterpart in std (or I am now aware of it), but `fill_zero` sounds fine to me. This way, you can leave `fill_with` name for the method taking a filling closure, as is also a [convention in std](https://doc.rust-lang.org/nightly/std/primitive.slice.html#method.fill_with).